examples/test_all.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+pushd $(dirname $0)
+
+for d in */ ; do
+    pushd $d
+    echo "Checking $d compiles"
+    RUSTFLAGS='-C link-arg=-s' cargo +nightly check --target wasm32-unknown-unknown --release
```

Comment:
I think try remove `+nightly`. So in travis there's two build, nightly-2019-09-03 and nightly (latest), both contain nightly rust only. but doing explicit "+nightly" will override in nightly-2019-09-03 build.
Doing two build is to tell a specific rust version that works, and let you know if it still works on latest nightly

Reply:
I don't think we need to test against `nightly-2019-09-03` at all. Going to switch it to latest nightly from nearcore

Reply:
There was a time near-bindgen doesn't pass with latest nightly and this was needed to make it pass. In future still likely new nightly breaks test so it's better to test with both a pinned version (2019-10-21 for nearcore now) and latest nightly.

Reply:
just saw you replace with stable. if stable works too that's better